.yamato/_projectcontext.yml

```diff
@@ -377,9 +377,9 @@ all_package_ci_project_2021.1:
         flavor: b1.large
     dependencies:
       -  .yamato/_projectcontext.yml#test_all_project_Win_2021.1
+      -  .yamato/_projectcontext.yml#test_all_project_Win_min_editor
```

Comment:
Are these supposed to be enabled? They don't work on Yamato (probably the min release field of the package.json files that needs to be updated in either this PR or a separate one) 
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/2021.1%252Fyamato%252Fpipeline-reorg/.yamato%252F_projectcontext.yml%2523test_all_project_OSX_min_editor/7304666/job/(log:Execution)

Reply:
https://unity.slack.com/archives/G0155CW8683/p1623741529000700 <- related

Reply:
You're right, I think these slipped back in in some of the latest commits. I've removed them again now,  thanks for catching it!